connectors/crowdstrike/connector/data_handler.py

```diff
@@ -52,6 +52,9 @@ def group_host_sensor_apps(applications):
             group_apps[agent_id].append(app)
     return group_apps
 
+def remove_duplicates(items):
+    """Remove any duplicate hosts from the array of hosts"""
```

Comment:
Would be nice to have some logging on this one. I'd log it as "debug" as it would be very chatty output.